mkosi/__init__.py

```diff
@@ -6293,6 +6292,7 @@ def find_output(args: argparse.Namespace) -> None:
     else:
         return
 
+    mkdirp_chown_current_user(args.output_dir, chown=args.chown)
```

Comment:
Are these (and the other additions like it) new or am I missing something? If they are new, could you put them in a separate commit?

Reply:
My bad, commits mixup.